ruby-pardot.gemspec

```diff
@@ -14,8 +14,8 @@ Gem::Specification.new do |s|
   s.required_rubygems_version = ">= 1.3.6"
   s.rubyforge_project         = "ruby-pardot"
 
-  s.add_dependency "crack"
-  s.add_dependency "httparty"
+  s.add_dependency "crack", "0.4.3"
+  s.add_dependency "httparty", "0.13.1"
```

Comment:
Need to explicitly define the versions due to incompatibilities with newer versions of Ruby (2.5+) and newer versions of these libraries. Locking to known good versions.